book/src/template_syntax.md

```diff
@@ -328,6 +328,15 @@ blocks from the base template with those from the child template. Inside
 a block in a child template, the `super()` macro can be called to render
 the parent block's contents.
 
+This is also why the `extends` tag doesn't support whitespace control:
```

Comment:
I feel like the "This" reference isn't that obvious. Maybe "Because top-level content from the child template is thus ignored, the `extends tag` ..."?

Reply:
It's better indeed. Updated!